android/app/src/main/java/com/microsoft/reactnativecodepush/CodePushConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.microsoft.reactnativecodepush;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+
+public class CodePushConfig {
```

Comment:
I would be in favor of completely removing this class. I don't think it adds enough value to exist separately from the `Codepush` class. I eventually want to refactor it out of the Obj-c implementation.